src/DynamoCoreWpf/Extensions/ViewLoadedParams.cs

```diff
@@ -40,6 +46,13 @@ public void AddSeparator(MenuBarType type, Separator separatorObj, int index = -
             AddItemToMenu(type, separatorObj, index);
         }
 
+        public event Action<NotifyCollectionChangedEventArgs> SelectionCollectionChanged;
+        private void OnSelectionCollectionChanged(object sender, NotifyCollectionChangedEventArgs notifyCollectionChangedEventArgs)
+        {
+            if (SelectionCollectionChanged != null)
+                SelectionCollectionChanged(notifyCollectionChangedEventArgs);
+        }
```

Comment:
Exposing the selection changed event through `ViewLoadedParams` so that it can register with and handled by `DMExtension`. See PR: https://git.autodesk.com/Dynamo/DynamoPro/pull/178